lua/telescope/pickers.lua

```diff
@@ -493,6 +493,14 @@ function Picker:find()
           log.warn("Finder failed with msg: ", msg)
         end
 
+        -- add an nvim_win_set_cursor to force the results win be redrawn
+        vim.schedule(function()
+          if not vim.api.nvim_win_is_valid(self.results_win) then
+            return
+          end
+          vim.api.nvim_win_set_cursor(self.results_win, vim.api.nvim_win_get_cursor(self.results_win))
```

Comment:
Would it be better to call `vim.cmd.redraw`?

```lua
vim.api.nvim_win_call(self.results_win, vim.cmd.redraw)
```

Reply:
I have tried `redraw!`. But it doesn't work either.

Reply:
Works for me.

```lua
        vim.schedule(function()
          vim.api.nvim_win_call(self.results_win, function()
            vim.cmd'redraw!'
          end)
        end)
```

Reply:
In fact even this works for me:

```lua
        vim.schedule(function()
          vim.api.nvim_win_call(self.results_win, function()
            vim.cmd.redraw()
          end)
        end)
```

Reply:
Oh, it works. But i don't know, what's the difference between
```
          vim.api.nvim_win_call(self.results_win, function()
            vim.cmd "redraw!"
          end)
```

and

```
vim.cmd "redraw!"
```

Reply:
just to make sure you used `nvim_win_call` and not just a simple `vim.cmd` right?

Reply:
`nvim_win_call` changes the context to the picker window so that window is redrawn.

Reply:
Ok. Update it using `redraw!`.